### ISSUE TITLE
yum install broken on manylinux/cibuildwheel used in main.yml

### ISSUE THREAD

**Issue by sarnold** [andreasvc/pyre2#16]
Apparently there was some recent churn due to centos EOL so the yum command is broken in your ci builds (right after your last tag build).  Supposedly this is "fixed" in the latest cibuildwheel release *but* now the ``yum`` command fails with ``No package re2-devel available``.  I tried bumping the manylinux image to latest but that also fails.  I've never used cibuildwheel or that other stuff before so I'm out of ideas.

**Comment by sarnold** [comment]
The fix includes a few things:

in build_wheels env:

```
    env:
      MANYLINUX2010_X86_64_TAG: "2020-12-03-912b0de"
```

update version of cibuildwheel:
```
        run: |
          python -m pip install cibuildwheel==1.7.1
```
update yum install:
```
          CIBW_BEFORE_ALL_LINUX: >
            yum -y -q --enablerepo=extras install epel-release
            && yum install -y re2-devel
```

**Comment by andreasvc** [comment]
Thanks for reporting. To be honest, I have never used cibuildwheel either, and I don't have access to a yum-based system. Could you perhaps submit a pull request with these fixes? Thanks in advance!

**Comment by sarnold** [comment]
Note this was fixed for manylinux wheels in previous PR and fixed again (for macos wheels) in #20
